src/tests/ftest/object/object_integrity.py

```diff
@@ -192,14 +194,56 @@ def test_invalid_tx_commit_close(self):
             self.log.info(str(excep))
             self.fail("##(3)Failed on close_tx.")
 
+        try:
+            self.container.abort_tx(invalid_transaction)
+            self.fail(
+                "##(4.1)Container.abort_tx passing with invalid handle")
+        except DaosApiError as excep:
+            self.log.info(str(excep))
+            self.log.info(
+                "==(4)Expecting failure: invalid Container.abort_tx.")
+            if expected_error not in str(excep):
+                self.fail(
+                    "##(4.2)Expecting error RC: -1002, but got {}."
+                    .format(str(excep)))
+
+        #Try to abort the transaction which already closed.
+        try:
+            self.container.abort_tx(new_transaction)
+            self.fail(
+                "##(5.1)Container.abort_tx passing with a closed handle")
+        except DaosApiError as excep:
+            self.log.info(str(excep))
+            self.log.info(
+                "==(5)Expecting failure: Container.abort_tx closed handle.")
+            if expected_error not in str(excep):
+                self.fail(
+                    "##(5.2)Expecting error RC: -1002, but got {}."
+                    .format(str(excep)))
+
+        #open another transaction for abort test
+        try:
+            new_transaction2 = self.container.get_new_tx()
+        except DaosApiError as excep:
+            self.fail("##(6.1)container get_new_tx failed: {}".format(excep))
+        self.log.info("==new_transaction2=     %s", new_transaction2)
+        self.ioreq.single_insert(c_dkey, c_akey, c_value, c_size,
+                                 new_transaction2)
+        try:
+            self.container.abort_tx(new_transaction2)
+            self.log.info("==(6)container.abort_tx test passed.")
+        except DaosApiError as excep:
+            self.log.info(str(excep))
+            self.fail("##(6.2)Failed on abort_tx.")
+
 
     @avocado.fail_on(DaosApiError)
     def test_single_object_validation(self):
         """
         Test ID: DAOS-707
         Test Description: Write Avocado Test to verify single data after
                           pool/container disconnect/reconnect.
-        :avocado: tags=all,object,full_regression,small,single_object,
+        :avocado: tags=all,object,full_regression,small,single_object
```

Comment:
unique test tag is missing.

Reply:
Thanks for the comment.
Please see the new commit for the fix.